docs/framework/backbone.md

```diff
@@ -0,0 +1,306 @@
+# Backbone Reference
+
+!!! failure "Deprecated"
+    CiviCRM no longer recommends using Backbone. This page is here primarily for archival purposes.
+
+## Background
+
+Backbone is a model-view (MV) framework for Javascript which follows a
+minimalist, plugin-oriented architecture: at its core, Backbone defines
+a minimal set of classes and utilities for building user-interfaces with
+object-oriented Javascript. Like the Drupal and jQuery communities, the
+Backbone community has a wide range of plugins to address missing
+functionality.
+
+## Examples
+
+Backbone is currently used in the following parts of CiviCRM:
+
+-   Profile Designer (v4.3+): Drag/drop interface for creating
+    profile forms. (At time of writing, usable with CiviSurvey)
+-   CiviVolunteer Extension (v4.4+): Drag/drop interface for managing
```

Comment:
ok so having a quick look at CiviHR github repo i can see https://github.com/civicrm/civihr/blob/e3d4768c37edc1292ed62bf7f6fcec82758ffb02/hrabsence/CRM/HRAbsence/Page/EmployeeAbsencePage.php#L93 that its still there and will check CiviVolunteer as well